src/components/Bartender.js

```diff
@@ -0,0 +1,47 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import actions from '../action_creators';
+import constants from '../constants';
+import moment from 'moment';
+import OpenEntryQueueContainer from './OpenEntryQueue';
+import _ from 'ramda';
+
+const OpenEntryQueue = createFactory(OpenEntryQueueContainer);
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+class Bartender extends Component {
+  componentWillMount() {
+    this.props.loadOrders();
+  }
+
+  render() {
+    const isOpen = _.pathEq(['entry', 'status'], constants.OPEN);
```

Comment:
Should the open status check be built into the `OpenEntryQueue` component, given its name?